tensorboard/webapp/persistent_settings/persistent_settings_module.ts

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {NgModule} from '@angular/core';
+
+import {PersistentSettingsConfigModule} from './persistent_settings_config_module';
+import {PersistentSettingsDataSourceModule} from './_data_source/persistent_settings_data_source_module';
+
+/**
+ * Persistent Settings module is responsible for persisting and loading settings
+ * from other features.
+ *
+ * For settings of the app, please refer to "settings" feature instead.
+ */
+@NgModule({
+  imports: [PersistentSettingsDataSourceModule],
+  providers: [PersistentSettingsConfigModule],
```

Comment:
is `PersistentSettingsModule` just a wrap for `PersistentSettingsConfigModule`?
but the usage is still using `PersistentSettingsConfigModule`? (e.g. `PersistentSettingsConfigModule.defineGlobalSetting` in `metrics_module.ts` in earlier pr version)

Reply:
`PersistentSettingsConfigModule` and `PersistentSettingsModule` are different. Former is a module that only provides settings _configuration_ and `PersistentSettingsModule` (really, effects that is exposed as part of `PersistentSettingsModule`) uses ConfigModule. I hope this helps.